[BUG] MKS SGEN_L LCD pin not correct for CR10_STOCKDISPLAY

### Description

MKS SGEN_L LCD pin not correct for CR10_STOCKDISPLAY

### Steps to Reproduce

<!-- Please describe the steps needed to reproduce the issue -->

1. Download Marlin 2.0
2. #define MOTHERBOARD BOARD_MKS_SGEN_L in Configuration.h
3. #define CR10_STOCKDISPLAY in Configuration.h
4. compile

**Expected behavior:** [What you expect to happen]

Should compile without any errors

**Actual behavior:** [What actually happens]

Compiling fails with the following error:
`/pins/lpc1768/pins_MKS_SGEN_L.h:195:28: error: 'P1_0' was not declared in this scope
`

### Fix
In **pins_MKS_SGEN_L.h** `#define LCD_PINS_RS   P1_0 ` needs to be changed to
`#define LCD_PINS_RS    P1_00`
There is a **0** missing

**Additional information**
Found in the [makerbase marlin repo](https://github.com/makerbase-mks/MKS-SBASE/blob/160580d0e9ddded5058e3af65baa55304e59e3b9/Firmware/Marlin-bugfix-2.0.x/Marlin/src/pins/pins_MKS_SGEN_L.h#L237)

--- a/Marlin/src/pins/lpc1768/pins_MKS_SGEN_L.h
+++ b/Marlin/src/pins/lpc1768/pins_MKS_SGEN_L.h
@@ -192,7 +192,7 @@
   #define BTN_ENC          P1_30
 
   #if ENABLED(CR10_STOCKDISPLAY)
-    #define LCD_PINS_RS    P1_0
+    #define LCD_PINS_RS    P1_00
 
     #define BTN_EN1        P0_18
     #define BTN_EN2        P0_15

